defensics/coap_main.py

```diff
@@ -43,7 +43,10 @@ def main():
     proxy = CoapProxy(DEVICE_ADDR, options.dev_id)
 
     automation = CoapAutomationHandler(proxy, tcp)
-    automation.start()
+    while (not automation.is_alive()):
+        automation = CoapAutomationHandler(proxy, tcp)
+        automation.start()
+        logging.debug('Proxy connected')
```

Comment:
This may be misleading. Information about proxy connection should probably be in proxy. Maybe something like "Automation started" would be better.